static/styles/image_upload_widget.scss

```diff
@@ -131,7 +131,7 @@
         right: 40px;
     }
 
-    .settings-page-upload-text {
+    .image-upload-text {
         top: 90px;
         right: 20px;
```

Comment:
Here, right should be 24px because the image width is 200px and the upload text ~151px. So the text is not well aligned.

Reply:
![Captura de tela de 2020-07-20 19-03-44](https://user-images.githubusercontent.com/15564640/87991767-40063600-cabd-11ea-8f16-978940825115.png)

Reply:
oh ok thanks Clara

Reply:
You're welcome! :)

Reply:
We should probably just replace this with more advanced alignment CSS that's scale invariant.